### ISSUE TITLE
[BUG] Cannot install on Arch

### ISSUE THREAD

**Issue by anquegi** [emacs-eask/cli#248]
Hello, I'm using manjaro a distro based on arch linux. 

Linux toni 6.6.30-2-MANJARO #1 SMP PREEMPT_DYNAMIC Wed May  8 17:46:43 UTC 2024 x86_64 GNU/Linux

When I try to install the package, it works, but  the resulting one do no start

```
    ~/learn/setup  git clone git@github.com:emacs-eask/cli.git                                                        ✔ 
S'està clonant a «cli»...
remote: Enumerating objects: 13560, done.
remote: Counting objects: 100% (2549/2549), done.
remote: Compressing objects: 100% (869/869), done.
remote: Total 13560 (delta 1986), reused 2058 (delta 1637), pack-reused 11011
S'estan rebent objectes: 100% (13560/13560), 6.01 MiB | 9.22 MiB/s, fet.
S'estan resolent les diferències: 100% (9225/9225), fet.
    ~/learn/setup  LC_ALL=C makepkg -sic                                                                              ✔ 
==> ERROR: PKGBUILD does not exist.
    ~/learn/setup  cd cli                                                                                          12 ✘ 
    ~/learn/setup/cli    master  LC_ALL=C makepkg -sic                                                             ✔ 
==> Making package: eask 0.9.8.bf4869c-1 (Tue May 21 10:32:37 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
==> Extracting sources...
==> Starting prepare()...

added 130 packages, and audited 131 packages in 5s

22 packages are looking for funding
  run `npm fund` for details

1 moderate severity vulnerability

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
==> Starting build()...

> @emacs-eask/cli@0.9.8 pkg-linux-x64
> pkg package.json -t node18-linux-x64 --output dist/eask

> pkg@5.8.1
==> Entering fakeroot environment...
==> Starting package()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "eask"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: eask 0.9.8.bf4869c-1 (Tue May 21 10:32:59 2024)
==> Installing package eask with pacman -U...
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (1) eask-0.9.8.bf4869c-1

Total Installed Size:  44.54 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                              [###########################################] 100%
(1/1) checking package integrity                                            [###########################################] 100%
(1/1) loading package files                                                 [###########################################] 100%
(1/1) checking for file conflicts                                           [###########################################] 100%
(1/1) checking available disk space                                         [###########################################] 100%
:: Processing package changes...
(1/1) installing eask                                                       [###########################################] 100%
:: Running post-transaction hooks...
(1/2) Arming ConditionNeedsUpdate...
(2/2) Refreshing PackageKit...
==> Cleaning up...
```

But after installing it doesnt work:

```
    ~  eask --version                                                                                                 ✔ 
Pkg: Error reading from file.
    ~  pamac search eask                                                                                            1 ✘ 
eask  0.9.8.bf4869c-1 [Instal·lats]                                                                                           
    CLI for building, running, testing, and managing your Emacs Lisp dependencies
```

**Comment by jcs090218** [comment]
Sorry, I don't use Arch. 🤔 

@Hi-Angel Do you have any thoughts?

**Comment by Hi-Angel** [comment]
Will look at it this evening

**Comment by Hi-Angel** [comment]
Should be fixed by https://github.com/emacs-eask/cli/pull/249

**Comment by anquegi** [comment]
> Should be fixed by #249

Applying patch on my system it worked, thanks
